### PR TITLE
Bug fixing some z-indexes

### PR DESCRIPTION
**Pull request by mat1th**
Hello, 
Thanks for your nice theme. I've added some bug fixes to your time. 

And I've also added a build setup in the README.md to build the the app.

### CODE REVIEW

**Comment by mat1th** [comment]
The bug I saw is no fixed;

File: src/scss/parts/_header.scss [hunk 1]
```diff
@@ -7,9 +7,13 @@
 // Général design
 //
 header#pm_header-desktop {
-	background: $color-header;
-	border-bottom: 1px solid darken($color-header, 5%);
-	z-index: 100;
+ background: $color-header;
+ /* Rectangle: */
+ background-image: -webkit-linear-gradient(bottom, $color-header, #46477F);
```
Comment (jeremiecook) [inline src/scss/parts/_header.scss]: I prefer to stick with flat header for now... I'll add  an option to modify header color or gradient from the main scss file soon. BTW you don't need to specify browser prefixes, they are handled by Gulp/PostCSS.

**Comment by mat1th** [comment]
Okay I've discart the changes on the header background gradient, and added a Contributors in the README.md.

**Comment by jeremiecook** [comment]
Thanks !

**Pull request merged**